src/index.html

```diff
@@ -5,8 +5,445 @@
+            <a href="#gadgets" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="page">
+      <div class="card page__card card--hover" data-qa="card">
+        <img
+          class="card__image"
+          src="/images/imac.jpeg"
+          alt="APPLE A1419 iMAC 27 Retine 5K Monoblock (MNED2UA/A)"
+        >
+
+        <h1 class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
+
+        <p class="card__code">
+          Product code: 195434
+        </p>
+
+        <div class="card__mark">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__mark_count">
+            Reviews: 5
+          </p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-text">
+            Price:
+          </p>
+
+          <p class="card__price-number">
+            $2,199
+          </p>
+        </div>
+
+        <a
+          href="#"
+          class="card__button"
+          data-qa="card-hover"
+        >
+          Buy
+        </a>
+      </div>
+
+      <div class="card page__card card--hover">
+        <img
+          class="card__image"
+          src="/images/imac.jpeg"
+          alt="APPLE A1419 iMAC 27 Retine 5K Monoblock (MNED2UA/A)"
+        >
+
+        <h1 class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
+
+        <p class="card__code">
+          Product code: 195434
+        </p>
+
+        <div class="card__mark">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__mark_count">
+            Reviews: 5
+          </p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-text">
+            Price:
+          </p>
+
+          <p class="card__price-number">
+            $2,199
+          </p>
+        </div>
+
+        <a
+          href="#"
+          class="card__button"
+          data-qa="card-hover"
+        >
+          Buy
+        </a>
+      </div>
+
+      <div class="card page__card card--hover">
+        <img
+          class="card__image"
+          src="/images/imac.jpeg"
```

Comment:
Replace the absolute path for the image source with a relative path: './images/imac.jpeg'. This is a recommendation